src/plugins/intel_cpu/src/nodes/experimental_detectron_generate_proposals_single_image.cpp

```diff
@@ -304,6 +180,12 @@ ExperimentalDetectronGenerateProposalsSingleImage::ExperimentalDetectronGenerate
     coordinates_offset = 0.0f;
 
     roi_indices_.resize(post_nms_topn_);
+    try {
+        nms_kernel_ = compileNMSKernel();
+    } catch (const InferenceEngine::GeneralError &e) {
+        DEBUG_LOG("Unable to create JIT version of GenerateProposals: ",
```

Comment:
Instead of throwing exception you could just return empty std::unique_ptr

Reply:
`compileNMSKernel()` method was removed in sake of `createPrimitive()` implementation.